config/testgrids/config.yaml

```diff
@@ -619,6 +619,14 @@ test_groups:
   gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-4.2
 - name: release-openshift-ocp-installer-e2e-aws-serial-4.2
   gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-serial-4.2
+- name: release-openshift-ocp-installer-e2e-azure-4.2
```

Comment:
Please put these additions (as well as any other openshift tests you want to include) in a separate file

```
test_groups:
 <all this stuff>
```